internal/services/workloads/workloads_sap_discovery_virtual_instance_resource.go

```diff
@@ -245,7 +245,7 @@ func (r WorkloadsSAPDiscoveryVirtualInstanceResource) Read() sdk.ResourceFunc {
 				}
 				state.Identity = pointer.From(identity)
 
-				props := &model.Properties
+				props := model.Properties
```

Comment:
Since this is now a pointer we should be nil checking this like we do in other resources:
```suggestion
				if props := model.Properties; props != nil {
					// set things into state here
				}
```

Reply:
Updated